Tests/CompatibilitySession.Tests.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Management.Automation
+using namespace System.Management.Automation.Runspaces
+
+Describe "Test the Windows PowerShell Compatibility Session functions" {
+
+    BeforeAll {
+        Import-Module -Force ..\WinCompatibilityPack\WinCompatibilityPack.psd1
+    }
+    
+    It "Make sure the <command> command exists" -TestCases @(
+        @{command = 'Initialize-WinSession'},
+        @{command = 'Add-WinFunction'},
+        @{command = 'Invoke-WinCommand'},
+        @{command = 'Get-WinModule'},
+        @{command = 'Import-WinModule'},
+        @{command = 'Compare-WinModule'},
+        @{command = 'Copy-WinModule'},
+        @{command = 'awinf'},
+        @{command = 'cpwinm'},
+        @{command = 'cwinm'},
+        @{command = 'gwinm'},
+        @{command = 'iwinc'},
+        @{command = 'iwinm'},
+        @{command = 'iwins'}
+        ) {
+            param($command)
+            Get-Command $command | Should -Not -BeNullOrEmpty
+        }
+
+    It "Calling Initialize-WinSession should return a valid session" {
+        Initialize-WinSession -PassThru | Should -BeOfType ([PSSession])
+    }
+
+    It "Multiple calls to Initialize-WinSession shouild return the same session" {
+        $first = Initialize-WinSession -PassThru
+        $second = Initialize-WinSession -PassThru 
+        $first | Should -Be $second
+    }
+
+    It "Get-WinModule should return a single value for PnpDevice" {
+        $info = Get-WinModule PnpDevice
+        $info | Should -Not -BeNullOrEmpty
+        $info.Name | Should -BeExactly PnpDevice 
```

Comment:
For clarity, please quote string literals in tests. `$info.Name | Should -BeExactly 'PnpDevice'`

Reply:
Done.